<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(127, 127, 127);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_CityName">
        <property name="font">
         <font>
          <family>Dubai medium</family>
          <pointsize>30</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 300 30pt &quot;Dubai medium&quot;;</string>
        </property>
        <property name="text">
         <string>City Country</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_Sunset">
          <property name="styleSheet">
           <string notr="true">font: 300 15pt &quot;Dubai&quot;;</string>
          </property>
          <property name="text">
           <string>Sunset: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QLabel" name="label_Temp">
          <property name="styleSheet">
           <string notr="true">font: 300 50pt &quot;Dubai Light&quot;;</string>
          </property>
          <property name="text">
           <string>0.00째C</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_minTemp">
          <property name="styleSheet">
           <string notr="true">font: 300 15pt &quot;Dubai&quot;;</string>
          </property>
          <property name="text">
           <string>min: 0.00째C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_maxTemp">
          <property name="styleSheet">
           <string notr="true">font: 300 15pt &quot;Dubai&quot;;</string>
          </property>
          <property name="text">
           <string>max: 0.00째C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Sunrise">
          <property name="styleSheet">
           <string notr="true">font: 300 15pt &quot;Dubai&quot;;</string>
          </property>
          <property name="text">
           <string>Sunrise: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QLabel" name="label_Icon">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/resources/img/11d.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_Description">
        <property name="styleSheet">
         <string notr="true">font: 300 20pt &quot;Dubai&quot;;</string>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_FeelsLike">
           <property name="styleSheet">
            <string notr="true">font: 300 15pt &quot;Dubai medium&quot;;</string>
           </property>
           <property name="text">
            <string>Feels like:  0.00째C</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Wind">
           <property name="styleSheet">
            <string notr="true">font: 300 15pt &quot;Dubai medium&quot;;</string>
           </property>
           <property name="text">
            <string>Wind speed: 0km/h</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Humidity">
           <property name="styleSheet">
            <string notr="true">font: 300 15pt &quot;Dubai medium&quot;;</string>
           </property>
           <property name="text">
            <string>Humidity: 0%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Search</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSearch">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(234, 234, 234);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Paris</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSearch()</slot>
 </slots>
</ui>
